templates/commands/specify.md

```diff
@@ -7,9 +7,23 @@ scripts:
 
 Given the feature description provided as an argument, do this:
 
-1. Run the script `{SCRIPT}` from repo root and parse its JSON output for BRANCH_NAME and SPEC_FILE. All file paths must be absolute.
-2. Load `templates/spec-template.md` to understand required sections.
-3. Write the specification to SPEC_FILE using the template structure, replacing placeholders with concrete details derived from the feature description (arguments) while preserving section order and headings.
-4. Report completion with branch name, spec file path, and readiness for the next phase.
+1. Run pre-specify hook if available (ignore errors):
+   - Windows: Try `.specify/hooks/pre-specify.ps1 "{ARGS}"` then `.specify/hooks/pre-specify "{ARGS}"`
```

Comment:
I am still on the fence about how we want to handle multi-script scenarios, that's why you see `{SCRIPT}` above that dynamically swaps the right script at package build time.

Thoughts?